flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/GenericWriteAheadSinkTest.java

```diff
@@ -122,15 +123,15 @@ protected void verifyResultsWhenReScaling(
         }
 
         Collections.sort(sink.values);
-        Assert.assertArrayEquals(list.toArray(), sink.values.toArray());
+        assertThat(sink.values.toArray()).containsExactly(list.toArray());
```

Comment:
```suggestion
        assertThat(sink.values).containsExactly(list);
```
Can't we go for a simpler solution here? :thinking:

Reply:
I found a bit different way 
```java
assertThat(sink.values).containsExactlyInAnyOrderElementsOf(list);
```
and there is no need to do sort one line eralier